### PR TITLE
use correct name of reduceVolume verb

### PR DESCRIPTION
**Pull request by clouseau**
Changes proposed in this pull request:

- fix name of reduceVolume verb to match name used in https://github.com/oVirt/vdsm/commit/a02fd4e9e2b1ed79bf52fb04f9f894934fba85e9

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @clouseau

This pull request needs review by: @sandrobonazzola

### CODE REVIEW

**Comment by rollandf** [comment]
:+1:

**Comment by rollandf** [comment]
@sandrobonazzola can you please take a look ?

File: source/release/4.2.0/index.html.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ packages from other repos.
...
- - [BZ 1168327](https://bugzilla.redhat.com/1168327) <b>Live Merge: optimize internal volume size</b><br>Feature: <br>Reduce (base) volume size to optimal after live merge.<br><br>Reason: <br>During live merge, the base volume is extended to accommodate merging the top volume into it.<br><br>Result: <br>A new verb, ReduceVolue, is now called at the of live merge to reduce (base) volume size to optimal.
+ - [BZ 1168327](https://bugzilla.redhat.com/1168327) <b>Live Merge: optimize internal volume size</b><br>Feature: <br>Reduce (base) volume size to optimal after live merge.<br><br>Reason: <br>During live merge, the base volume is extended to accommodate merging the top volume into it.<br><br>Result: <br>A new verb, reduceVolume, is now called at the of live merge to reduce (base) volume size to optimal.
```
Comment (sandrobonazzola) [inline source/release/4.2.0/index.html.md]: Looks like doc-text in https://bugzilla.redhat.com/show_bug.cgi?id=1168327 completely changed. Now is: "After live merge, the base volume is reduced to its optimal size." This is what will be written here next time I'll regenerate the release notes.

Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: The new content has been already reviewed by doc team so if "After live merge, the base volume is reduced to its optimal size." is not enough, please comment in the bug.

**Pull request merged**